src/app/models/organization.ts

```diff
@@ -0,0 +1,11 @@
+import { Item } from './item';
+
+export class Organization extends Item {
+  public indentName (baseIndentation: number) {
```

Comment:
Can be used by other types of items, so put the function `indentName` dans be moved into `item.ts` file

Reply:
I'm not sure yet if it makes sense for other types of items (it's my "no too-early generalization" principle :)), so that's why I kept it in the Organization model. Also, moving it in `item.ts` would make it available in the `User` model as well, which doesn't make sense.

Reply:
ok ;)